src/app/gui/map/mapservice.js

```diff
@@ -18,7 +18,6 @@ const WFSProvider = require('core/layers/providers/wfsprovider');
 const olhelpers = require('g3w-ol/g3w.ol').helpers;
 const {getScaleFromResolution, getResolutionFromScale} = require('core/utils/ol');
 const ControlsFactory = require('gui/map/control/factory');
-const StreetViewService = require('gui/streetview/streetviewservice');
 const ControlsRegistry = require('gui/map/control/registry');
```

Comment:
@volterra79 can you confirm that it was only a typo? (i.e. unusued variable)

Reply:
Yes. Non more used